apps/api/src/.example.env

```diff
@@ -38,3 +38,5 @@ VERCEL_REDIRECT_URI=http://localhost:4200/auth/login
 VERCEL_BASE_URL=https://api.vercel.com
 
 FF_IS_TOPIC_NOTIFICATION_ENABLED=true
+
+STORE_NOTIFICATION_CONTENT=true
```

Comment:
Can we make it false by default? Also adding this to env validator?

Reply:
Do you mean on the .example.env?

Reply:
No in this file: apps/api/src/config/env-validator.ts where the schema is defined for env variables and defaults also provided

Reply:
Oh got ya, I will update shortly! 👍